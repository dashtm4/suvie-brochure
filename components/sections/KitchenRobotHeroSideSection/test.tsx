import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { KitchenRobotHeroSideSection } from '.'

describe('<KitchenRobotHeroSideSection />', () => {
  it('should renders correct <KitchenRobotHeroSideSection />', () => {
    const wrapper = shallow(<KitchenRobotHeroSideSection />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
