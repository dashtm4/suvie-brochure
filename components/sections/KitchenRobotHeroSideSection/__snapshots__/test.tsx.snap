// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<KitchenRobotHeroSideSection /> should renders correct <KitchenRobotHeroSideSection /> 1`] = `
<div
  className="flex-1 text-center md:order-none"
>
  <Typography
    className="mb-2"
    theme="dark"
    variant="subtitle2"
  >
    <span
      className="font-normal"
    />
    <br
      className="hidden md:block"
    />
     
    <span
      className="font-medium"
    />
  </Typography>
  <Typography
    className="font-normal"
    theme="dark"
  />
</div>
`;
